src/core_modules/capture-core/components/Pages/MainPage/EventsList/Pagination/ListPagination.container.js

```diff
@@ -0,0 +1,24 @@
+// @flow
```

Comment:
_This comment applies for the whole file_

Replaced this src/core_modules/capture-core/components/Pages/MainPage/EventsList/Pagination/withData.js
with a container. Looked to me that this change moves this component closer to our style.